Shared/Tests.Shared/TestHelpers.cs

```diff
@@ -86,7 +86,7 @@ public static void RunEventLoop(int milliseconds)
 
         public static void RunEventLoop()
         {
-            RunEventLoop(1);
+            RunEventLoop(100);
```

Comment:
I'm not sure why this is here - it's supposed to be in master already. Anyway, it controls how much time we idle for when waiting for notifications. 1 ms worked well on iOS, but on Android it seems too short and in some of our tests, the notification will not be delivered, thus giving us false negatives.